2 failing settings/avatar functional tests

```
× firefox on any platform - settings/avatar - go to settings with an email NOT selected to see change link should not see one (0.579s)
Error: a.change-avatar should not be present
  at Command.<anonymous>  <tests/functional/lib/helpers.js:196:19>
  at runCallback  <node_modules/intern/node_modules/leadfoot/Command.js:526:31>
  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:543:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at Object.run [as _onImmediate]  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at processImmediate [as _immediateCallback]  <timers.js:345:15>
  at Command.then  <node_modules/intern/node_modules/leadfoot/Command.js:542:10>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:195:12>
  at runCallback  <node_modules/intern/node_modules/leadfoot/Command.js:526:31>
  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:543:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at Object.run [as _onImmediate]  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at processImmediate [as _immediateCallback]  <timers.js:345:15>
  at Command.then  <node_modules/intern/node_modules/leadfoot/Command.js:542:10>
  at Test.registerSuite.go to settings with an email NOT selected to see change link should not see one [as test]  <tests/functional/avatar.js:107:10>
  at start.then.finally.self.timeElapsed  <node_modules/intern/lib/Test.js:211:24>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/node_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:317:4>
  at Object.run [as _onImmediate]  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at processImmediate [as _immediateCallback]  <timers.js:345:15>
× firefox on any platform - settings/avatar - go to settings with an email NOT selected to see change link should not see text link (0.607s)
AssertionError: expected true to be false
  at Assertion.<anonymous>  <node_modules/intern/node_modules/chai/chai.js:572:10>
  at Assertion.Object.defineProperty.get  <node_modules/intern/node_modules/chai/chai.js:3515:29>
  at Function.assert.isFalse  <node_modules/intern/node_modules/chai/chai.js:2204:31>
  at Command.<anonymous>  <tests/functional/avatar.js:122:18>
  at runCallback  <node_modules/intern/node_modules/leadfoot/Command.js:526:31>
  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:543:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at Object.run [as _onImmediate]  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at processImmediate [as _immediateCallback]  <timers.js:345:15>
  at Command.then  <node_modules/intern/node_modules/leadfoot/Command.js:542:10>
  at Test.registerSuite.go to settings with an email NOT selected to see change link should not see text link [as test]  <tests/functional/avatar.js:121:10>
  at start.then.finally.self.timeElapsed  <node_modules/intern/lib/Test.js:211:24>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/node_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:317:4>
  at Object.run [as _onImmediate]  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at processImmediate [as _immediateCallback]  <timers.js:345:15>
```

--- a/tests/functional/avatar.js
+++ b/tests/functional/avatar.js
@@ -21,10 +21,8 @@ define([
   var AVATAR_CHANGE_URL_AUTOMATED = config.fxaContentRoot + 'settings/avatar/change?automatedBrowser=true';
 
   var PASSWORD = 'password';
-  var EMAIL_AVATAR_AB_PREFIX = 'avatarAB-';
   var email;
   var client;
-  var emailAvatarAb;
 
   function testIsBrowserNotifiedOfAvatarChange(context) {
     return FunctionalHelpers.testIsBrowserNotified(context, 'profile:change', function (data) {
@@ -67,7 +65,6 @@ define([
 
     beforeEach: function () {
       email = TestHelpers.createEmail();
-      emailAvatarAb = EMAIL_AVATAR_AB_PREFIX + TestHelpers.createEmail();
 
       client = new FxaClient(AUTH_SERVER_ROOT, {
         xhr: nodeXMLHttpRequest.XMLHttpRequest
@@ -94,70 +91,34 @@ define([
         .end();
     },
 
-    'go to settings with an email NOT selected to see change link should not see one': function () {
-      return this.remote
-        .get(require.toUrl(SETTINGS_URL))
-
-        .setFindTimeout(intern.config.pageLoadTimeout)
-
-        .findByCssSelector('.avatar-view')
-        .end()
-
-        // this link should not be present on the page at all
-        .then(FunctionalHelpers.noSuchElement(this, 'a.change-avatar'));
-    },
-
-    'go to settings with an email NOT selected to see change link should not see text link': function () {
-      return this.remote
-        .get(require.toUrl(SETTINGS_URL))
-        .setFindTimeout(intern.config.pageLoadTimeout)
-
-        .findByCssSelector('.avatar-view')
-        .end()
-
-        // this link will be present on the page, but not visible to the user.
-        .setFindTimeout(0)
-        .findByCssSelector('p.change-avatar-text a').isDisplayed()
-        .then(function (isDisplayed) {
-          assert.isFalse(isDisplayed);
-        })
-        .end();
-    },
-
     'go to settings with an email selected to see change link then click on avatar to change': function () {
       var self = this;
-      return signUp(self, emailAvatarAb)
-        .then(function () {
-          return self.remote
-            .get(require.toUrl(SETTINGS_URL))
+      return self.remote
+        .get(require.toUrl(SETTINGS_URL))
 
-            // go to change avatar
-            .findByCssSelector('a.change-avatar')
-              .click()
-            .end()
+        // go to change avatar
+        .findByCssSelector('a.change-avatar')
+          .click()
+        .end()
 
-            // success is going to the change avatar page
-            .findById('avatar-options')
-            .end();
-        });
+        // success is going to the change avatar page
+        .findById('avatar-options')
+        .end();
     },
 
     'go to settings with an email selected to see change link then click on text link to change': function () {
       var self = this;
-      return signUp(self, emailAvatarAb)
-        .then(function () {
-          return self.remote
-            .get(require.toUrl(SETTINGS_URL))
+      return self.remote
+        .get(require.toUrl(SETTINGS_URL))
 
-            // go to change avatar
-            .findByCssSelector('p.change-avatar-text a')
-              .click()
-            .end()
+        // go to change avatar
+        .findByCssSelector('p.change-avatar-text a')
+          .click()
+        .end()
 
-            // success is going to the change avatar page
-            .findById('avatar-options')
-            .end();
-        });
+        // success is going to the change avatar page
+        .findById('avatar-options')
+        .end();
     },
 
     'visit gravatar with gravatar set': function () {

